nil/internal/execution/zerostate.go

```diff
@@ -125,29 +127,21 @@ func DumpMainKeys(fname string) error {
 	return err
 }
 
-func LoadMainKeys(fname string) error {
+func LoadMainKeys(fname string) (*ecdsa.PrivateKey, []byte, error) {
```

Comment:
Why do we return private key as a proper type and public key as sequence of bytes?

Reply:
Yes, will be fixed in next PR.